tracer/src/Datadog.Tracer.Native/debugger_tokens.h

```diff
@@ -20,6 +20,36 @@ using namespace trace;
 
 namespace debugger
 {
+enum ProbeType {NonAsyncMethod, AsyncMethod, NonAsyncLine};
```

Comment:
When I looked through the usages of these I noticed you're not always qualifying these with `ProbeType::`, so I'd suggest making them more explicit.
```suggestion
enum ProbeType {MethodProbeInNonAsyncMethod, MethodProbeInAsyncMethod, LineProbeInNonAsyncMethod};
```
Or, just keep this as  `enum ProbeType { LineProbe, MethodProbe }` and pass a boolean around for `isAsyncMethod`. But whatever works.

Also, where's `LineProbeInAsyncMethod`? How come we don't need that?

Reply:
Line probe will be addressed in the upcoming PR